pkg/controller/schedule_controller.go

```diff
@@ -370,15 +370,26 @@ func getBackup(item *api.Schedule, timestamp time.Time) *api.Backup {
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: item.Namespace,
 			Name:      fmt.Sprintf("%s-%s", item.Name, timestamp.Format("20060102150405")),
-			Labels: map[string]string{
-				"ark-schedule": item.Name,
-			},
 		},
 	}
 
+	// add schedule labels and 'ark-schedule' label to the backup
+	addLabelsToBackup(item, backup)
+
 	return backup
 }
 
+func addLabelsToBackup(item *api.Schedule, backup *api.Backup) {
+
+	labels := item.Labels
+	if labels == nil {
+		labels = make(map[string]string)
+	}
+	labels["ark-schedule"] = item.Name
+
+	backup.ObjectMeta.Labels = labels
```

Comment:
You could also do `backup.Labels = labels` (we tend not to include the embedded intermediate structs when referencing fields).